flink-kubernetes/src/main/java/org/apache/flink/kubernetes/kubeclient/decorators/InitJobManagerDecorator.java

```diff
@@ -98,7 +103,7 @@ private Container decorateMainContainer(Container container) {
                 .withImagePullPolicy(kubernetesJobManagerParameters.getImagePullPolicy().name())
                 .withResources(requirements)
                 .withPorts(getContainerPorts())
```

Comment:
Here for example, the user might wanna configure some extra ports for queryable state or so.